scap/src/main.rs

```diff
@@ -36,12 +38,12 @@ fn main() {
         show_cursor: true,
```

Comment:
Is there a specific reason we've updated the output_type, source_rect values in `options`. If not, let's keep the older ones.